### ISSUE TITLE
Dashboard: Filter out Checklist items

### ISSUE THREAD

**Issue by dwertheimer** [NotePlan/plugins#498]
I tend to use Tasks for things I really need to do, and checklists for lesser things that I don't want nagging me
So I now see the checklists on my Dashboard and I am not sure I want to see them.
What if you add a switch to toggle showing checklist items like this one you already hve?
Is the switch state remembered (that would be nice)?

<img width="321" alt="Screen Cap 2023-11-21 at 13 56 15@2x" src="https://github.com/NotePlan/plugins/assets/8949588/512ebc14-8ee3-47b4-984c-83a59b7951ac">

I guess that one switch could do what I want, but I am not sure what the algorithm is that it uses, because 

I have some checklist items like this:
<img width="256" alt="Screen Cap 2023-11-21 at 13 59 07@2x" src="https://github.com/NotePlan/plugins/assets/8949588/2d9aed5f-5231-487b-894e-e53535d0bb13">

When I flip the switch, I get the first one, but not the prioritized ones.
<img width="462" alt="Screen Cap 2023-11-21 at 14 00 12@2x" src="https://github.com/NotePlan/plugins/assets/8949588/98b5fa64-eb87-46d8-ae32-a8c2dee1be33">

and in Overdue I have a bunch of tasks, two of which are priority:
<img width="356" alt="Screen Cap 2023-11-21 at 14 00 35@2x" src="https://github.com/NotePlan/plugins/assets/8949588/297496f8-43ea-4600-89d6-7a008c0147b9">

but the filter button leaves only one of them:
<img width="416" alt="Screen Cap 2023-11-21 at 14 01 06@2x" src="https://github.com/NotePlan/plugins/assets/8949588/a5e74761-c2e8-4890-bd93-4a1fa15a7320">

This could all be due to the fact that I don't always put the !!! at the front of the line, because I find that impedes readability in my notes greatly. 

So maybe it's working as designed, but I'd still like a way to hide checklists when I don't want to see them.

**Comment by jgclark** [comment]
OK, there are several different issues here.

1. I have added a new setting for 0.7.3 to ignore checklist items everywhere in the Dashboard. I decided for now to not make it a visible checkbox as a) they're much more work, and b) I can't imagine most users wanting to actually change it once set to their preference.

2. Eduard says priority markers have to come at the start of a line, and that's what the built-in themes do. So my helpers work to that standard as well.

3. My filtering-by-priority logic leaves only the highest present priority item(s) in a given section. So what you're seeing above is what I'd expect.
